pact/ledger.pact

```diff
@@ -168,7 +175,7 @@
     (enforce-verify-manifest manifest)
     (policy::enforce-init
       { 'id: id, 'supply: 0.0, 'precision: precision, 'manifest: manifest })
-    (emit-event (TOKEN id))
+    (emit-event (TOKEN id precision policy))
```

Comment:
I don't _think_ we want to have `manifest` in the event as manifest's can be _fat_. That said, putting them in now would stress-test the events API and trip issues that we'll see later as we move to Sever Sent Events.